src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -346,6 +365,7 @@ type RepliesProps = {
     isParentPending?: boolean,
     isRepliesLoading?: boolean,
     mentionSelectorContacts?: SelectorItems<>,
+    onEdit: Function,
```

Comment:
Can we type this in arrow notation to be more explicit with parameters and return type?

Reply:
Yes, good call. This should only ever be `OnCommentEdit`.